### ISSUE TITLE
[dataclasses] I3Direction docs (Trac #1730)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1722]
https://wiki.icecube.wisc.edu/index.php/I3Direction

These docs shouldn't be in a wiki, but instead part of the project/class documentation.

Migrated from https://code.icecube.wisc.edu/ticket/1730
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:47", 
    "description": "https://wiki.icecube.wisc.edu/index.php/I3Direction\n\nThese docs shouldn't be in a wiki, but instead part of the project/class documentation.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067167842669", 
    "component": "combo core", 
    "summary": "[dataclasses] I3Direction docs", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-06-07T13:37:44", 
    "milestone": "", 
    "owner": "olivas", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160721T16:05:17: olivas changed status from "new" to "accepted"

**Comment by icecube-trac** [comment]
Trac update at 20160729T12:57:54: 
* kjmeagher changed status from "accepted" to "closed"
* kjmeagher commented:

> In changeset 148897:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="148897"
> bring I3Direction docs into sphinx documentation closes #1722
> ```


* kjmeagher changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:47: 
* olivas changed milestone from "June 2016" to ""
* olivas commented:

> Milestone June 2016 deleted
